ocs_ci/ocs/constants.py

```
@@ -2671,6 +2671,11 @@
 RHCOS = "RHCOS"
 
 # Scale constants
+IBM_STORAGE_SCALE_NAMESPACE = "ibm-spectrum-scale"
+REMOTE_CLUSTER = "RemoteCluster"
+SCALE_FILESYSTEM = "Filesystem"
+SCALE_GUI_USER = "fdf_user_apr16"
+SCALE_GUI_PASSWORD = "passw0rd"
```

Ruff (0.15.14)

[error] Possible hardcoded password assigned to: "SCALE_GUI_PASSWORD"

(S105)